### DISCUSSION TITLE
[Paid Plan]-No artifacts were found - Publishing to Testflight

### DISCUSSION THREAD

**Discussion by ferdiangunawan** [codemagic-ci-cd/codemagic-docs#1704] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Hello, I can't publish my IPA to Appstore because no artifacts were found. Thank you
<img width="814" alt="Screenshot 2023-05-24 at 14 59 48" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/49526345/82831c18-2607-461a-aa67-98b842385e1d">

### Expected results

Processing to upload to Appstore

### Actual results

== Gathering artifacts ==
No artifacts were found

### Build id (optional)

646d9390ec00b5a406069f25

**Comment by icarusdust** [comment, marked as answer]
Hi @ferdiangunawan, please note that you should contact our team through the chat widget in the UI for quicker response as your team is on a paid plan. 
As for the issue, you did not have the artefacts sections set correctly but I can see that you have already configured it. Let us know in case of any further questions in Intercom.

**Reply by ferdiangunawan** [reply]
Hi, thank you for the reply!
